modules/phase_field/src/kernels/ACGrGrPoly.C

```diff
@@ -18,20 +18,15 @@ InputParameters validParams<ACGrGrPoly>()
 
 ACGrGrPoly::ACGrGrPoly(const InputParameters & parameters) :
     ACBulk(parameters),
+    _ncrys(coupledComponents("v")),
+    _vals(_ncrys),
+    _vals_var(_ncrys),
     _mu(getMaterialProperty<Real>("mu")),
     _gamma(getMaterialProperty<Real>("gamma_asymm")),
     _tgrad_corr_mult(getMaterialProperty<Real>("tgrad_corr_mult")),
-    _has_T(isCoupled("T")),
```

Comment:
This change is functionally equivalent though! I just got rid of the boolean and check the pointer for NULL.